tests/test-xkarpenter/main.k

```diff
@@ -0,0 +1,492 @@
+                        }
+                    }
+                }
+                iamv1beta1.RolePolicyAttachment {
+                    metadata = {
+                        annotations = {
+                            "crossplane.io/composition-resource-name" = "instanceNodeRoleEKSPolicy"
+                        }
+                        generateName = "configuration-aws-eks-karpenter-"
+                        labels = {
+                            "crossplane.io/composite" = "configuration-aws-eks-karpenter"
+                        }
+                    }
+                    spec = {
+                        deletionPolicy = "Delete"
+                        forProvider = {
+                            policyArn = "arn:aws:iam::aws:policy/AmazonEKSWorkerNodePolicy"
+                            roleSelector = {
+                                matchControllerRef = True
+                                matchLabels = {
+                                    role = "karpenter"
+                                }
+                            }
+                        }
+                        managementPolicies = ["*"]
+                        providerConfigRef = {
+                            name = "default"
+                        }
+                    }
+                }
+                iamv1beta1.RolePolicyAttachment {
+                    metadata = {
+                        annotations = {
+                            "crossplane.io/composition-resource-name" = "instanceNodeRoleSSMPolicy"
+                        }
+                        generateName = "configuration-aws-eks-karpenter-"
+                        labels = {
+                            "crossplane.io/composite" = "configuration-aws-eks-karpenter"
+                        }
+                    }
+                    spec = {
+                        deletionPolicy = "Delete"
+                        forProvider = {
+                            policyArn = "arn:aws:iam::aws:policy/AmazonSSMManagedInstanceCore"
+                            roleSelector = {
+                                matchControllerRef = True
+                                matchLabels = {
+                                    role = "karpenter"
+                                }
+                            }
+                        }
+                        managementPolicies = ["*"]
+                        providerConfigRef = {
+                            name = "default"
+                        }
+                    }
+                }
+                iamv1beta1.InstanceProfile {
+                    metadata = {
+                        annotations = {
+                            "crossplane.io/composition-resource-name" = "instanceProfile"
+                        }
+                        generateName = "configuration-aws-eks-karpenter-"
+                        labels = {
+                            "crossplane.io/composite" = "configuration-aws-eks-karpenter"
```

Comment:
same for all labels crossplane.io/composite - these labels gets added by crossplane after apply